### DISCUSSION TITLE
If there is no repository in the list, description is not visible in the stars tab.

### DISCUSSION THREAD

**Discussion by SimDaSong** [community/community#8736] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/35736920/145707254-af6272e6-34ee-4196-a2d2-17f336fa9cc1.png)
https://github.com/SimDaSong?tab=stars

I didn't add a repository to the todo, interview list, and I added a repository to the Node.js, lectures list.

In the stars tab, you can see the description of Node.js, lectures list, but not the description of todo, interview list.

I'm not sure if it's an intended feature, but if it's an intended feature, I hope I can also see the description of the empty list on the stars tab.

Because I am not good at English, I used the translator, so the explanation may be insufficient.
If you need more explanation, please leave a comment.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up, this is a bug! Our team has taken a look and will release a fix for this soon.
